packages/xstate-vue/src/useFsm.ts

```diff
@@ -0,0 +1,26 @@
+import { ref, onBeforeMount, onBeforeUnmount, Ref } from '@vue/composition-api';
+import { StateMachine, EventObject, interpret } from '@xstate/fsm';
+import { AnyEventObject } from 'xstate';
+
+export function useFsm<TC, TE extends EventObject = AnyEventObject>(
```

Comment:
IMHO it would be better to match `@xstate/react` naming here - so this would be a `useMachine`. It will be easier for people to educate people about it.

Reply:
Ok, looking at your useMachine - I could have misunderstood something. Could you describe what's the purpose of this hook? It seems like both your hooks are responsible for calling `interpret` and this confuses me.

Reply:
Okay, I'll change that then.

Reply:
I just copied the structure of the [React FSM useMachine hook](https://github.com/davidkpiano/xstate/blob/master/packages/xstate-react/src/fsm.ts), but called it useFsm to show the difference that when using `useFsm` you are actually relying on `@xstate/fsm` and not on `@xstate`.

But as I said in my description, I could also of course name it `useMachine` as well. It wouldn't get an export in `index.ts` though then.

Reply:
Ah, ok - I get it now. Was reviewing late and didn't spot this one is actually using `@xstate/fsm`. I think we need to settle on a solution for https://github.com/davidkpiano/xstate/pull/847 and then we'll be able to just follow the suite in here in regards to supporting both `xstate` & `@xstate/fsm`